changelog.md

```diff
@@ -1,3 +1,79 @@
+    * material.{colormap|...|metallicmap}.rw attribute: changed rotation center from (0.5, 0.5) to (0,0).
+* Bugfixes:
+    * readStringTable, readFloatTable: Fixed a crash when reading a non-exisitng table (Linux only).
+    * setupProjection operation:
+      * Fixed a bug where widthOffset and heightOffset were not corretly considered when a uv scaling was defined for the material (for example material.colormap.su).
+      * Fixed a bug where offset values where not considered for projection axes scope.zx.
+    * CGA Compiler:
+      * Fixed incorrect handling of the @Hidden annotation for import statements. Note: In the UI of older versions of CityEngine or client apps using an older version of CityEngine SDK, hidden imports will not be hidden in RPKs created with CityEngine 2024.0.
+      * Fixed an bug where annotations were not assigned to import statements but to subsequent statements.
+
+## Built-In Codecs
+* GLTF Encoder:
+  * Now correcly converts colors to linear space.
+  * Adapted texture trafo rotation center change: from (0.5, 0.5) to (0,0).
+* GLTF Decoder:
+  * Now correcly converts colors from linear space.
+  * Adapted texture trafo rotation center change: from (0.5, 0.5) to (0,0).
+* Unreal Encoder:
+  * Adapted texture trafo rotation center change: from (0.5, 0.5) to (0,0).
+* USD Encoder:
+  * Fixed "triangulate faces with holes" functionality - in some cases tiny gaps in the connectivity were created.
+  * Added support for texture map trafos. 
+* USD Decoder:
+  * Added support for texture map trafos. 
+* IFC Encoder:
+  * Fixed "triangulate faces with holes" functionality - in some cases tiny gaps in the connectivity were created.
+  * Fixed support for units in IFC4 (was fixed to meters). 
+* OBJ Encoder:
+  * Fixed "triangulate faces with holes" option - in some cases tiny gaps in the connectivity were created.
+* Shape Buffer Encoder:
+  * Fixed "triangulate faces with holes" option - in some cases tiny gaps in the connectivity were created.
+  * Added support for "doubleSided" material attribute.
+* Shape Buffer Decoder:
+  * Added support for "doubleSided" material attribute.
+* FBX Encoder:
+  * Fixed "triangulate faces with holes" option - in some cases tiny gaps in the connectivity were created.
+* FBX Decoder:
+  * Added support for normal maps.
+* CGAMat Decoder:
+  * Fixed a rare crash when decoding materials.
+* Collada Encoder:
+  * Fixed "triangulate faces with holes" option - in some cases tiny gaps in the connectivity were created.
+* Alembic Encoder:
+  * Fixed "triangulate faces with holes" option - in some cases tiny gaps in the connectivity were created.
+* TIFF Decoder:
+  * Fixed a rare deadlock (which was introduced in SDK 3.2.9903). 
+
+## Misc Changes and Fixes
+* In the CGA and the PRT reference colorspaces have been clarified. Colors are always in (non-linear) sRGB space.
```

Comment:
Just occurred to me: did we add this to the CE help (CGA Ref) as well?

Reply:
yes, in both the material attribute and the color() pages